sql_export_mail/views/sql_export_view.xml

```diff
@@ -36,6 +37,7 @@
                             nolabel="1"
                             colspan="2"
                             domain="[('model_id', '=', 'sql.export')]"
+                            groups="base.group_system"
```

Comment:
I would put the cron on the group level (or both). Else, the user  without group will just see the group name with nothing else and it is kind of strange.
Apart from this, LGTM, thanks